### PR TITLE
chore: moved exit codes page in "Fundamentals of Tact"

### LINKED ISSUE #328: Make exit codes link more visible
We should move it from the bottom of the `Going places` section in the table of contents somewhere else. Maybe `Fundamentals of Tact`

### PR DESCRIPTION
**Pull request by novusnota**
...and also slightly updated the index page to encourage people to search and seek answers themselves first, before they go in Tact chats. That won't work on everyone, of course, but it's important to at least mention that we have the search box :)

Closes #328

### CODE REVIEW

File: pages/index.mdx [hunk 1]
```diff
@@ -141,7 +144,9 @@ Alternatively, take a look at the following broader sections:
 
 ### Feeling a bit uncomfortable? [#next-3]
 
-If you ever get stuck, don't hesitate to reach out to Tact's flourishing community:
+If you ever get stuck, try searching — the search box is right at the top of the documentation. There is also a handy `Ctrl + K` shortcut to quickly focus and start the search as you type.
```
Comment (anton-trunov) [inline pages/index.mdx:147]: maybe use `<kbd>` for the keyboard shortcut?

Comment (novusnota) [reply pages/index.mdx:147]: Maybe, but it doesn't get underlined like inline code block does:

image

Comment (anton-trunov) [reply pages/index.mdx:147]: image
github makes it prettier then :)

Comment (anton-trunov) [reply pages/index.mdx:147]: this is the rendered version of this source: `<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Space</kbd>`

Comment (novusnota) [reply pages/index.mdx:147]: One downside there is that such notation gets split by lines because of the +

**Pull request merged**